Editing pass over cover letter + manuscript (#46)
chaynes2019/AvidaGeneDupe: tex/text/body/methods.tex

\subsection{Evolution trials}

> We conducted experiments using a customized version of Avida v2.14.0 that supports our slip mutation ablation treatments \citep{Ofria:2009avida}.

Reviewer: ```suggestion
We conducted experiments using a custom version of Avida v2.14.0, extended to support our slip mutation ablation treatments \citep{Ofria:2009avida}.
```
